website/theme.config.tsx

```diff
@@ -0,0 +1,38 @@
+import { Rabbit, Package } from "lucide-react";
+
+export default {
+  logo: (
+    <span
+      style={{
+        display: "flex",
+        color: "#0E7490",
+      }}
+    >
+      <Rabbit style={{ marginRight: "5px" }} />
+      +
+      <Package style={{ marginRight: "5px" }} />
+      =
+      <strong style={{ marginRight: "5px" }}>Hutch</strong>Documentation
```

Comment:
would add `marginLeft: 5px` here as well, because now it's so close to the "=" on its left